### ISSUE TITLE
[Linux] Symlinks shown as files in the native file selector

### ISSUE THREAD

**Issue by SunderlandJames** [a85/POSTMan-Chrome-Extension#5740]
## App Details:
```
Postman for Linux
Version 6.5.3
linux 4.4.0-141-generic / x64
```

## Feature Request:
Recognise Symlinks for directories as directories rather then files to allow traversal for import/export and other related features.

As an example on my machine, I have an SSD and a HDD with my boot and home drive on the SSD. Due to space concerns i have all my home directories (music, pictures, documents, downloads) as Symlinks to the HDD.

Currently Postman cannot recognise these symlinks as directories meaning I can not export to/import from the standard downloads location or even documents location.
![Screenshot showing symlinks being detected as files](https://i.imgur.com/Kp2cjWf.jpg)

**Comment by harryi3t** [comment]
@SunderlandJames 
I was not able to reproduce this. Let me know if I missed something. Here's what I have tried:

- Created a new VM for Ubuntu 16. (Looking at the Linux kernel version (4.4.0), looks like you are on Ubuntu 16)
- Attached a separate hard drive
- Created a shortcut on my desktop to a folder on another hard drive
- Tried exporting a collection

![image](https://user-images.githubusercontent.com/5207331/68081841-4888e480-fe3a-11e9-830b-077c65e092c7.png)

Since the issue looks like in the native file selector, could you try using some other application like VSCode or anything that can import/export?

**Comment by numaanashraf** [comment]
Thank you for your issue @SunderlandJames! 

We haven't gotten a response to our questions in our comment above. With only the information that is currently in the issue, we don't have enough information to take action. I'm going to close this but don't hesitate to reach out if you have or find the answers we need, we'll be happy to reopen the issue.

**Comment by trodix** [comment]
![Screenshot_20200326_223246](https://user-images.githubusercontent.com/34584317/77698997-dadc8100-6fb1-11ea-80ae-a012c0daa163.png)

Same issue here, on Kubuntu 19.10 (linux 5.3.0-42-generic / x64 ) and Postman **Version 7.20.0**.
**I can't access to my symlinks** while browsing for a file. See the _workspace_ symlink on the screenshot above.

**Comment by trodix** [comment]
A think a found where the problem is.

If we install Postman with snap or in my case with flatpak, the sandbox permissions don't allow us to use our default system's document manager. In this case, the application sand-boxed don't have access to anything else than the default flatpak or snap permissions.

If we install Postman by using the .tar.gz archive from [Postman website](https://www.postman.com/downloads/ ) and execute Postman/app/Postman, then we have the default system file manager and we have full access of our files and symlinks.

I had the same issue with others applications installed with snap or flatpak, like discord and chromium-browser, I did the same, I installed the .deb packages and the problem was solved.

**In conclusion, the problem is snap and flatpak permissions, not Postman itself.**

**Comment by numaanashraf** [comment]
Thanks for letting us know @trodix. We'll look into whether there is a way to grant these permissions in Postman's snap manifest.

**Comment by Moongazer** [comment]
> If we install Postman with snap or in my case with flatpak, the sandbox permissions don't allow us to use our default system's document manager.

Thanks a lot for pointing it out, we just run into the same issue! Snap is really the worst invention the Ubuntu creators ever had -,- Always problems, doesn't matter with which application. On top of that the huge package sizes and than new updates every 2-3 days (like Signal app)... its so shit :being-angry:

**Comment by lcapalbo** [comment]
I'd just experienced this issue with the flatpak version, and reported it to https://github.com/flathub/com.getpostman.Postman/issues/168

This is what I found out: I was able to build the flatpak without the sandbox to the user home (by editing the `com.getpostman.Postman.yaml` manifest setting the `finish-args:` from `--filesystem=home` to `--filesystem=host`) and that fixed the issue for me, but I understand there's a security concern behind the default to home access only.   This is still experienced as a bug on the flatpak version with no clue to the final user, at least someone with as little flatpak knowledge as myself..

I was suggested to use https://flathub.org/apps/details/com.github.tchx84.Flatseal to tweak the flatpak permissions (after installing the default one) and that did the trick in a cleaner way

**Comment by ijash** [comment]
Just wanted to add, if you use [flatseal](https://github.com/tchx84/flatseal), which is a permission utility for flatpak, toggling this solves the problem:

![image](https://user-images.githubusercontent.com/30673286/153717215-62b3a81f-511d-4044-aef0-2bac4682333a.png)

Select Postman and just scroll down on the right pane until you find **FIle System**, and toggle **All system files**
